src/lib/libraries/sounds.json

```diff
@@ -676,7 +676,7 @@
     {
         "name": "C2 Sax",
         "md5": "ea8d34b18c3d8fe328cea201666458bf.wav",
-        "sampleCount": 7349,
+        "sampleCount": 8129,
         "rate": 22050,
         "format": "adpcm",
```

Comment:
Is it worth converting these to regular .wavs so they can be decoded by the browser?

Reply:
this is a reasonable question - it's a tradeoff of file size vs js decoding time I guess- ADPCM is smaller but takes a little time to decode.  the other issue is that we don't generally save at 22kHz (i.e. sounds are always saved at 44kHz, except very long sounds are now saved at 22kHz, thanks to a recent update), and we no longer save in adcpm format. so any updates we make to library asset sounds will make them increase in size by 2x or more.